task_netcore/task4/Client/src/modules/AccountForm/actions/index.js

```diff
@@ -0,0 +1,75 @@
+import { SHOW_ERROR_MESSAGE, CLEAR_ERROR_MESSAGE, REQUESTED_AUTHENTICATION, REQUESTED_AUTHENTICATION_FAILED , REQUESTED_AUTHENTICATION_SUCCEEDED } from './types';
+import { webApiRoutes } from '../../../Constants';
+import axios from 'axios';
+import { isAuth } from '../../Header/actions/index';
+import { SessionService } from '../../../Services/SessionService';
+import { applicationRoutes } from '../../../Constants';
+
+export const showErrorMessage = (errorMessage) => {
+    return {
+        type: SHOW_ERROR_MESSAGE,
+        errorMessage
+    }
+}
+
+export const clearErrorMessage = () => {
+    return {
+        type: CLEAR_ERROR_MESSAGE
+    }
+}
+
+export const requestAuthentication = () => {
+    return {
+      type: REQUESTED_AUTHENTICATION
+    }
+  };
+  
+  export const requestAuthenticationSuccess = (token, user) => {
+    return {
+      type: REQUESTED_AUTHENTICATION_SUCCEEDED,
+      token,
+      user
+    }
+  };
+  
+  export const requestAuthenticationError = (errorMessage) => {
+    return {
+      type: REQUESTED_AUTHENTICATION_FAILED,
+      errorMessage
+    }
+  };
+  
+  export const axiosLogin = (dispatch, user, history) => {
+    dispatch(requestAuthentication());
+
+    return axios.post(webApiRoutes.loginRoute, user)
+      .then(response => {
+        dispatch(requestAuthenticationSuccess(
+          SessionService.setItem('token', response.data),
```

Comment:
You can create an object, { token: value, userName: value }. This will help you to remove duplicates of SessionService.setItem() function calls.  Also, you can create a helper with methods:  
1) to stringify json object into string (to decrease readability of elements of these object in SessionStorage) 
2) for parsing this string back to object. 
Then you can call this object everywhere like an object with fields instead of two variables.